### DISCUSSION TITLE
Error en MPBricks con react native

### DISCUSSION THREAD

**Discussion by aixavilla** [mercadopago/sdk-js#157] [Q&A] [answered]
Al instanciar el sdk de Mercado Pago da un error, porque no encuentre el archivo index.js de la ruta @mercadopago/sdk-js/index.js el cual esta llamando en el package.json 

@mercadopago/sdk-js/package.json

{
  "name": "@mercadopago/sdk-js",
  "version": "0.0.2",
  "description": "Mercadopago SDK JS",
  "main": "index.js",
  "module": "dist/index.js",
  "jsnext:main": "dist/index.js",
  "types": "dist/index.d.ts",
  "typings": "dist/index.d.ts",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/mercadopago/sdk-js.git"
  },
  "bugs": {
    "url": "https://github.com/mercadopago/sdk-js/issues"
  },
  "scripts": {
    "build": "rm -rf ./dist && tsc -b ./tsconfig.json",
    "dev": "tsc -c -w",
    "test": "node test/test.js",
    "pretest": "npm run build"
  },
  "devDependencies": {
    "ts-loader": "^9.4.2",
    "tslib": "^2.4.1",
    "typescript": "^4.9.4"
  },
  "author": "MercadoPago (https://www.mercadopago.com/developers)",
  "license": "Apache-2.0"
}

efectivamente el archivo index.js no existe en el directorio mencionado. 

Error de consola: 
Metro has encountered an error: While trying to resolve module `@mercadopago/sdk-js` from file `/node_modules/@mercadopago/sdk-react/mercadoPago/initMercadoPago/index.js`, the package `/node_modules/@mercadopago/sdk-js/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/node_modules/@mercadopago/sdk-js/index.js`. Indeed, none of these files exist:

  */node_modules/@mercadopago/sdk-js/index.js(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json)
  * /node_modules/@mercadopago/sdk-js/index.js/index(.native|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx|.ios.js|.native.js|.js|.ios.jsx|.native.jsx|.jsx|.ios.json|.native.json|.json): /node_modules/metro/src/node-haste/DependencyGraph.js (272:17)

  270 |         }
  271 |         if (error instanceof InvalidPackageError) {
> 272 |           throw new PackageResolutionError({
      |                 ^
  273 |             packageError: error,
  274 |             originModulePath: from,
  275 |             targetModuleName: to

**Comment by AleFossati** [comment, marked as answer]
Hello @aixavilla, sorry for inconvenience, but this library was designed for web integrations and thus is not ready to work with React Native.
